kolibri/core/auth/management/commands/register.py

```diff
@@ -0,0 +1,92 @@
+import sys
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+from morango.certificates import Certificate
+from requests import exceptions
+
+from kolibri.core.auth.utils import confirm_or_exit
+from kolibri.core.auth.utils import get_facility
+from kolibri.core.utils.portal import claim
+
+
+class Command(BaseCommand):
+    help = "Allows registering of facilities which have been synced to a portal server."
```

Comment:
which -> that

Reply:
In terms of naming, we use "register" here, but "claim" on KDP. For consistency and clarity, perhaps unifying them? I agree that `claim` sounds a bit funny on the client side, so perhaps sticking with `register` and updating the KDP server naming accordingly? And maybe being more explicit with the name too, e.g. `registerfacility`?